src/components/backend-ai-storage-proxy-list.ts

```diff
@@ -42,6 +43,7 @@ import './lablup-progress-bar';
 export default class BackendAIStorageProxyList extends BackendAIPage {
   @property({type: String}) condition = 'running';
   @property({type: Array}) agents;
+  @property({type: Object}) spinner;
```

Comment:
According to the recent commit from main branch, these names has been updated.
Please fetch the recent version of the main branch.